portfolio/src/main/webapp/comment-form.js

```diff
@@ -54,12 +64,28 @@ function formatComments(container, comments) {
 
 /** Return a paragraph element that contains a user comment */
 function addComment(comment) {
+  const secElement = document.createElement('section');
+  secElement.appendChild(addCommentText(comment));
+  secElement.appendChild(addCommentImage(comment));  
+  return secElement; 
+}
+
+/** Return a paragraph element that contains a user comment */
+function addCommentText(comment) {
   const pElement = document.createElement('p');
   pElement.innerText = comment.email + ': ' + comment.text; 
   pElement.className = 'user-comment';
   return pElement; 
 }
 
+/** Return a paragraph element that contains a user comment */
+function addCommentImage(comment) {
+  const imgElement = document.createElement('img');
```

Comment:
Nit, you can also create `<img>` tag in JavaScript using the `Image` constructor

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image

Reply:
adjusted